maestro/config/config_builder.go

```diff
@@ -86,7 +86,7 @@ var k8sOtelCollector = `
             "containers": [
               {
                 "name": "otel-collector",
-                "image": "otel/opentelemetry-collector-contrib:0.91.0",
+                "image": "orb-community/otelcol-orb-maestro@latest",
```

Comment:
I'd recommend to avoid using `latest` but stick to specific version (once we have them)